CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). If you introduce breaking changes, please group them together in the "Changed" section using the **BREAKING:** prefix.
 
+## [Unreleased]
+
+### Added
+
+- Added SAS content navigator. You are now able to browse, edit, create, delete, and run files on a SAS server using a Viya connection.
```

Comment:
Oh, just remembered our last meeting suggested to add issue or pr numbers to changelog

Reply:
Changes look good to me.